src/components/LandingPage/Header.tsx

```diff
@@ -83,53 +83,132 @@ const MobileNav = styled.div`
     left: 0;
     top: 76px;
     width: 100%;
+    padding-left: 1.25rem;
     background: var(--white);
     flex-basis: 100%;
-    display: flex;
     flex-direction: column;
-    align-items: center;
+    align-items: start;
     justify-content: center;
     border-bottom: 1px solid var(--footer-divider);
-    padding-bottom: 12px;
-    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.05);
-    ${(props) => {
-        if (props.active) {
-            return `display: flex`;
-        } else {
-            return `display: none`;
-        }
-    }}
+    padding-bottom: 0.75rem;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.05);
+    padding-top: 1rem;
+    display: ${(props) => (props.active ? 'flex' : 'none')};
     z-index: 1;
 `;
+const Arrow = styled.i`
+    float: right;
+    border: solid #000000;
+    border-width: 0 2px 2px 0;
+    margin-top: 0.875rem;
+    margin-right: 1.75rem;
+    display: inline-block;
+    padding: 0.25rem;
+    transform: ${(prop) => (!prop.show ? 'rotate(45deg)' : 'rotate(-135deg)')};
+    -webkit-transform: ${(prop) =>
+        !prop.show ? 'rotate(45deg)' : 'rotate(-135deg)'};
+`;
 
-const MobileMenuItem = styled(Link)`
-    color: var(--dark-text);
-    font-size: 16px;
-    font-weight: 600;
-    position: relative;
-    height: 37px;
-    margin: 8px 0;
-    display: flex;
-    align-items: center;
-    text-decoration: none;
-    font-family: Source Sans Pro;
+const MobileMenuItem = styled.div`
+    font-family: IBM Plex Sans;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 31px;
+    letter-spacing: 0.01em;
+    text-align: left;
+    margin-bottom: 1.5rem;
+    display: block;
+    color: black;
+    float: left;
+    width: 99%;
 `;
 
-const NavBar = ({}) => {
+const MobileDropdown = styled.div`
+    display: ${(prop) => (!prop.show ? 'none' : 'block')};
+    color: black;
+    margin-left: 1.5rem;
+    margin-top: 1.5rem;
+`;
+const DropdownItem = styled(Link)`
+    display: block;
+    color: black;
+    font-size: 19px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 25px;
+    letter-spacing: 0.01em;
+    text-align: left;
+    &:not(:last-child) {
+        margin-bottom: 1.25rem;
+    }
+    ${(prop) =>
+        prop.disabled ? 'pointer-events: none;opacity: 0.5;color:grey;' : ''};
+`;
+
+const SubMenu = styled.div`
+    position: absolute;
+    display: ${(prop) => (!prop.show ? 'none' : 'flex')};
+    margin-top: 1rem;
+    flex-direction: column;
+    padding: 1.25rem;
+    border: 1px solid #d6d6d6;
+    border-radius: 6px;
+    background: #ffffff;
+`;
+const SubItem = styled(Link)`
+
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: 0.20000000298023224px;
```

Comment:
Why is this letter spacing so precise?

Reply:
well i copy those from figma...

Reply:
@pimato Why is this letter-spacing so precise :P